apps/codeforafrica/src/lib/data/blockify/posts.js

```diff
@@ -11,6 +11,13 @@ async function posts(block, api, context) {
   const featured = featuredStory ? formatPost(featuredStory, primaryTag) : null;
   const options = {
     ...query,
+    ...(featured && {
+      where: {
+        slug: {
+          not_equals: featured.slug,
```

Comment:
Let's make sure the `slug` field has [`index: true`](https://payloadcms.com/docs/fields/text#config) if we're going to be using it in queries.